CONTRIBUTING.md

```diff
@@ -45,6 +45,36 @@ To run code linter, use:
 npm run lint
 ```
 
+## Commit Messages
+
+Commit messages should follow the Semantic Commit Messages format.
```

Comment:
is there a link for "Semantic Commit Messages" that we can link to?

Reply:
No good link that I can refer to